extensions/amc-score-calculator/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "amc-score-calculator",
+  "title": "AMC Score Calculator",
+  "description": "Calculates AMC scores for 8, 10, and 12",
+  "icon": "command-icon.png",
+  "author": "ryanccn",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "eight",
+      "title": "AMC 8",
+      "subtitle": "",
+      "description": "Calculate AMC 8 scores",
```

Comment:
Maybe you can provide more detailed description, something like "Get your AMC 8 score from number of correct answers"

Reply:
Same in other two commands